docs/interactions.md

```diff
@@ -307,7 +307,7 @@ soundness condition.
 ### Total Interactions Trace Height Constraints
 
 We also add another constraint related to the bits of soundness for the LogUp procedure. For this, we set the trace
-height coefficients to be the number of interactions on the corresponding AIR and the threshold to be $2{^30}$. This
+height coefficients to be the number of interactions on the corresponding AIR and the threshold to be $2^{30}$. This
```

Comment:
This actually just changed to also be `p`: https://github.com/openvm-org/stark-backend/pull/52